src/controllers/index.js

```diff
@@ -1,5 +1,9 @@
 import UsersController from './users';
 import ResetPasswordController from './resetPassword';
+import AccommodationController from './accommodationController';
+import RoomController from './roomController';
 
-
-export { UsersController, ResetPasswordController };
+export {
+  UsersController, ResetPasswordController,
+  AccommodationController, RoomController
```

Comment:
why are you exporting these controllers here?

Reply:
Regle isn't tracking the controllers when exported from api